src/app/core/services/products.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ProductsService } from './products.service';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
Follow the styleguide. Validate all changes

https://angular.io/guide/styleguide#import-line-spacing

"**Their**" should come before "**Ours**"

![screen shot 2019-01-31 at 15 15 17](https://user-images.githubusercontent.com/9499109/52129534-6a9dce00-2640-11e9-9565-3a6691a18351.png)